src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/NotificationsType.php

```diff
@@ -40,13 +40,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('show_notifs_new_orders', SwitchType::class, [
-                'required' => true,
+                'label' => $this->trans('Show notifications for new orders', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('This will display notifications when new orders are made in your shop.', 'Admin.Advparameters.Help'),
```

Comment:
Here we can remove the help text, I think.

Reply:
Removed all 3 help texts

Reply:
Thank you!